### ISSUE TITLE
Support for latest SeaMonkey please

### ISSUE THREAD

**Issue by mdlueck** [protz/Manually-Sort-Folders#174]
Greetings,

It was suggested to use this extension as Folderpane Tools ( https://addons.thunderbird.net/en-US/seamonkey/addon/folderpane-tools/ ) is no longer supported, does not work any longer, with the latest SeaMonkey (Mozilla Suite).

I am seeking something to adjust the order of several email accounts, each with its own separate folder structure.

**Comment by itagagaki** [comment]
I installed SeaMonkey for the first time after reading this message. I don't know if add-ons for ThunderBird are available for SeaMonkey, and in fact, I couldn't install the current version of MSF.

I don't even know how to make it compatible with SeaMonkey, but I will look into it. I don't even know if I can or can't.
Please don't get your hopes up and wait patiently.

**Comment by mdlueck** [comment]
Greetings, Mozilla Suite / SeaMonkey is where Mozilla got the source code from to build the email client ThinderBird. Very similar... just that it has a browser also bundled... what became Firefox.

In the past I have had success with an add-on developer stating to hack their installation file, state that it is also SeaMonkey compatible, and their add-on worked without issue. You might get lucky.

**Comment by protz** [comment]
This would only be worth it if Seamonkey's Messenger codebase is up-to-date with what is in Thunderbird. Last I checked (but that was a few years ago), Seamonkey was so far behind Thunderbird that there was little hope of writing an addon that worked for both programs.

**Comment by mdlueck** [comment]
Please double check current. A couple of small releases ago there was an effort made to sync up TBird and SM code bases.

**Comment by itagagaki** [comment]
I tried a little. @protz was completely right.
But it looks like there is a possibility that I can make SeaMonkey-specific version based on past revision of MSF.
I'm thinking of creating a SeaMonkey branch.

**Comment by mdlueck** [comment]
Thank you so much for considering a SeaMonkey branch! :-)

**Comment by itagagaki** [comment]
MSF for SeaMonkey is now available.
[https://addons.thunderbird.net/ja/seamonkey/addon/manually-sort-folders-sm/](https://addons.thunderbird.net/ja/seamonkey/addon/manually-sort-folders-sm/)

Try it out.

**Comment by itagagaki** [comment]
Development complete

**Comment by mdlueck** [comment]
Terrific.... the extension indeed loads.

I went to the Tools \ Manual Sort Folders, and the only thing populated the first time was Local Folders. I was puzzled. I exited and went back to the same window.... THEN all my other accounts were populating in the list.

Indeed, switching Account #3 to the #2 slot, restart, swapped the accounts, and they swapped back as expected.

Bravo! Thank you very much for the SM port!
